### PR TITLE
doc: effectively joining the lxd group requires to log out/in

### PR DESCRIPTION
**Pull request by simondeziel**
I'm on the fence as to suggest to log out and back in or use `newgrp`.

### CODE REVIEW

File: doc/installing.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ If you want the current user to be able to interact with the LXD daemon, add it
 getent group lxd | grep -qwF "$USER" || sudo usermod -aG lxd "$USER"
 ```
 
+```{note}
+If your user was not already a member of the `lxd` group, you need to log out and back in for the change to take effect.
+```
+
```
Comment (ru-fu) [inline doc/installing.md:88]: Even though it's only one sentence, we should reuse it in the different locations. In this case, I think it's easiest to add it to `doc/substitutions.yaml` and then include it with `{{must_start_new_session}}` (or similar).

**Comment by ru-fu** [comment]
> I'm on the fence as to suggest to log out and back in or use `newgrp`.

We have both here: https://documentation.ubuntu.com/lxd/en/latest/installing/#manage-access-to-lxd
We could say that you need to make sure the new group is applied by either logging out and back in or using newgrp.

File: doc/installing.md (hunk 1 above)
Comment (simondeziel) [reply doc/installing.md]: Interesting mechanism I wasn't aware of, thanks.

**Review by tomponline — APPROVED** [review]
Thanks!

**Pull request merged**